Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+
+ESDC's policy and program development are done "in the open", over the Internet, enabling active participation with stakeholders whether they be citizens, 3rd party partners like banks, employers, or other jurisdictions.
+
+Technology choices **adopt open standards with preference for Open Source solutions** in order to avoid vendor lock-in, enable interoperability, and collaboration.
+Technology choices does not take a simple look as "COTS *or* Custom", rather choices look at fit for purpose which means "COTS *and* Custom": COTS for commodities and Custom for specialization. COTS is defined as per [US DoD FAQ](https://dodcio.defense.gov/Open-Source-Software-FAQ/#Q:_Is_open_source_software_commercial_software.3F_Is_it_COTS.3F)
+
+**Business Rules are seen as institutional knowledge** and are managed [internally by ESDC's staff](https://18f.gsa.gov/2018/10/09/implementing-rules-without-rules-engines/), outside of proprietary systems to avoid vendor lock-in and assure long term sustainability of Canadians core benefits systems.
+
+## 3. Digital-first and digitally enabled by design
+
+All of [ESDC's services](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) are delivered through technology with a large majority running on the public cloud to seek benefits of [rapid elasticity, broad internet access, and up to date security patching](https://doi.org/10.6028/NIST.SP.800-145). This includes up to Protected B data.
+Platform as a Service (PaaS, such as containers) and Software as a Service (SaaS) delivery cloud models ia preferred, leaving Infrastructure as a Service (IaaS) on exception basis only.
+
+ESDC's IT Solutions can only "talk" to each other via **APIs** allowing channel-agnostic services and communications.
+ESDC's API are all published on the Government of Canada's **API Store**, enabling the private industry and the public to build mobile applications, websites, applications, and voice assistant compatibility devices, to deliver a variety of user experiences to citizens.
+
+ESDC's IT solutions provide the flexibility needed to empower end-users with the choice of sharing personal data, bringing any privacy concerns in the hands of the end-user as opposed to ESDC making decisions for them.
+
+ESDC prefers **open source software** and products that adopt **open standards** over proprietary ones.
+The adoption of Open source software is the main vendor locking mitigation strategy.
+The adoption of open standards is the main interoperability enabler strategy.
+
+Legacy technologies like mainframe COBOL have been replaced by using a strangled approach as opposed to a big bang one.
```

Comment:
using a strangled approach
==>
by implementing [the strangler pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/strangler)

Reply:
Maybe "are replaced" instead?